example/android/app/src/main/AndroidManifest.xml

```diff
@@ -11,7 +11,8 @@
       android:allowBackup="false"
       android:theme="@style/AppTheme"
       android:networkSecurityConfig="@xml/network_security_config"
-      android:usesCleartextTraffic="true">
+      android:usesCleartextTraffic="true"
+      android:requestLegacyExternalStorage="true">
```

Comment:
Wondering if we should be using the new permission system for this.. for now it's fine but..